README.md

```diff
@@ -11,72 +11,135 @@
-| $ `karma COMMAND ../../karma.conf.js [--once]`  | Test the local package, where COMMAND is e.g. `start` or `run`
-| $ `rollup -c ../../rollup.config.js`  | Build the package into the local `dist/`
+  Imported namespace is available *only* as `tko`.
+
+* CommonJS <a id="imports-cjs"></a>
+
+  ```js
+  const ko = require("@tko/build.reference/dist/index.cjs")
+  ```
+
+* MJS <a id="imports-mjs"></a>
+
+  ```js
+  import ko from "@tko/build.reference/dist/index.mjs"
+  ```
+
+  `ko` is default export so feel free to change import name on your desired.
 
+> **Note**: you can check imported TKO version by looking at its namespace attribute `.version<String>` in runtime.
 
-#### Testing with `yarn test` and `yarn watch`
 
-The `yarn test` and `yarn watch` commands can be used in the root directory, where it will run across all tests, or alternatively in any `packages/*/` directory to run tests
-specific to that package.
+### Sites <a id="sites"></a>
 
-Optional arguments to `yarn test` include:
+Currently there are 2 points of interest which further would be considered a wiki:
+- https://tko.io - wiki specifically for TKO (*not Knockout 4*)
+- https://brianmhunt.github.io/knockout - Work In Profress wiki/docs for Knockout
+
+
+## Using the Monorepo <a id="monorepo"></a>
+
+The default package manager for monorepo currently is `npm` with the addition of [Lerna](https://lerna.js.org/docs/getting-started) as a viable choice for monorepo management.
+
+First thing at getting familiar with the project monorepo is to look at `./Makefile` at project root. Since its a separate file spanning entire repo, we will cover only the most commonly used commands:
+
+| Command | Effect |
+| ------- | ------ |
+| $ `git clone git@github.com:knockout/tko` | Clone the repository.
+| $ `make` | Install all dependencies, build *reference* and *compatibility* versions of TKO at `builds/*/dist/*` without any transpilation. It is the `default` makefile behavior calling `all` directive.
+| $ `make clean` | empties build directories `.../dist/*` and `package-lock.json`'s inside of every `package/*` and `builds/*`
+| $ `npm i` | Install local node packages and link tko modules
+| $ `make test` | Run all tests. See below.
+| $ `make publish-unpublished` | Bump versions and publish to npm registry
+
+In each individual `packages/*/` directory, you can also run (presuming `esbuild` and `karma` are installed globally):
+
+| Command | Effect |
+| --- | --- |
+| $ `make test` | Test the local package, abstraction of `npx karma start ../../tools/karma.conf --once`
+| $ `make` | Build the package into the local `dist/`
 
-- `--sauce` — use Sauce Labs to test a variety of platforms; requires an account at Sauce Labs and `SAUCE_USERNAME` and `SAUCE_ACCESS_KEY` to be set in the environment.
-- `--noStartConnect` — Do not start a new Sauce Connect proxy instance for every
-test; requires that Sauce Connect be already running.
 
+### Test suit <a id="tests"></a>
 
-#### `visual.html`
```

Comment:
Removed this section entirely since it looks like karma does not produce this file